far/vmenu.cpp

```diff
@@ -3296,7 +3296,7 @@ int VMenu::FindItem(int StartIndex, string_view const Pattern, unsigned long lon
 
 			if (Flags&LIFIND_EXACTMATCH)
 			{
-				if (starts_with_icase(strTmpBuf, Pattern))
+				if (equal_icase(strTmpBuf, Pattern))
```

Comment:
Let's just do `if (strTmpBuf == Pattern)` here.
"Exact match" should mean exact.
We can add case-insensitive flags later if needed.

Reply:
Done

Reply:
Looks good, pls update build and changelog.

Reply:
Updated